package.json

```diff
@@ -5,10 +5,14 @@
   "main": "skin-deep.js",
   "typings": "skin-deep.d.ts",
   "scripts": {
-    "test": "mocha",
     "lint": "eslint . --max-warnings 0",
     "coverage": "istanbul cover _mocha --",
-    "doc": "doctoc README.md"
+    "doc": "doctoc README.md",
+    "test": "npm run react:14 && mocha && npm run react:15 && mocha && npm run react:155 && mocha",
+    "react:clean": "rimraf node_modules/react node_modules/react-dom node_modules/react-addons-test-utils react-test-renderer react-addons-create-fragment @types/react create-react-class prop-types",
```

Comment:
The test suite already runs on multiple react versions, using the environment matrix feature of travis CI

See `.travis.yml` and `install-relevant-react.sh` for details.